concepts/case/about.md

```diff
@@ -3,11 +3,11 @@
 [Case][case] (often referred to as switch in other languages) is a form of control expression like if-else.
 Case allows for chaining of multiple if-else-if statements and can be more readable and also allows for powerful constructs.
```

Comment:
Shouldn't these also use backticks? Are you drawing a distinction between the concept of a case and a syntactical `case`?